.github/workflows/ci_dockers.yml

```diff
@@ -73,11 +73,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.7", "3.9"]
-        pytorch_version: ["1.8", "1.11"]
         include:
           # the config used in '.azure-pipelines/gpu-tests.yml'
-          - {python_version: "3.9", pytorch_version: "1.10"}
+          - {python_version: "3.7", pytorch_version: "1.8", cuda_version: "11.1"}
+          - {python_version: "3.7", pytorch_version: "1.10", cuda_version: "11.1"}
```

Comment:
Only one config is used for gpu tests. so this line should go after the comment.

Same comment in the nightly file

Reply:
> Only one config is used for gpu tests. so this line should go after the comment.

We'll be updating PyTorch version from `1.8` to `1.10` and to `1.11` eventually, so I updated the lines to have the two prospective versions there.